index.html

```diff
@@ -0,0 +1,798 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="initial-scale=1.0" />
```

Comment:
- [ ] Fix the linting error `The 'viewport' meta element 'content' attribute value should contain 'width=device-width'.`

add `width=device-width` to the content attribute.